cdk/eks/lib/stacks/eks-stack.ts

```diff
@@ -18,6 +18,7 @@ interface EksStackProps extends StackProps {
   cloudwatchAddonRoleProp: RoleProps,
   rdsClusterEndpoint: string,
   rdsSecurityGroupId: string,
+  tag: string
```

Comment:
can we use a more specific name like awsApplicationTag? Tag seems too generic. AWS resources can have other tags not related to myApplication.